### PR TITLE
Check the Account table before returning data

### PR DESCRIPTION
**Pull request by JericoFX**
**Pull Request Description**

_Please include a brief description of the changes you have made such as why you decided to implement this or what this changes._

This change will check if the table has that value and then return it

**Features**:
_Please include a list of features/fixes that this Pull Request contains._
- fix(client/lang): wrong translation
- feat(web/index): implemented new button 
 Feature
**Pull Request Checklist**:
* [ X] Have you followed the guidelines in our Contributing document and Code of Conduct?
* [ X] Have you checked for any open pull requests?
* [ ] Have you tested the changes yourself and with another testing subject (-cl2 works too) -- Tested in a online compiler
* [ ] Would you like to be included as a contributor?

### CODE REVIEW

File: shared/config_sv.lua [hunk 1]
```diff
@@ -1,7 +1,12 @@
 Config = {
     Spawn = vec(-75.5010, -818.9702, 326.1753, 221.4246),
     SaveTime = 10000,
-    Accounts = { cash = 0, bank = 0, black = 0, tebex = 0  },
+    Accounts = {
```
Comment (BombayV) [inline shared/config_sv.lua:4]: Any reason of using brackets?

Comment (JericoFX) [reply shared/config_sv.lua:4]: It's out of habit, I can remove the strings if requested

Comment (BombayV) [reply shared/config_sv.lua:4]: Yeah. Leave it like it was. At the moment there's no need for brackets.

Comment (JericoFX) [reply shared/config_sv.lua:4]: done

**Pull request merged**